exchanges/cryptodotcom/cryptodotcom_wrapper.go

```diff
@@ -0,0 +1,953 @@
+	}
+	acc, err := account.GetHoldings(cr.Name, creds, assetType)
+	if err != nil {
+		return cr.UpdateAccountInfo(ctx, assetType)
+	}
+	return acc, nil
+}
+
+// GetFundingHistory returns funding history, deposits and
+// withdrawals
+func (cr *Cryptodotcom) GetFundingHistory(ctx context.Context) ([]exchange.FundHistory, error) {
+	withdrawals, err := cr.GetWithdrawalHistory(ctx)
+	if err != nil {
+		return nil, err
+	}
+	deposits, err := cr.GetDepositHistory(ctx, currency.EMPTYCODE, time.Time{}, time.Time{}, 0, 0, 0)
+	if err != nil {
+		return nil, err
+	}
+	resp := make([]exchange.FundHistory, 0, len(withdrawals.WithdrawalList)+len(deposits.DepositList))
+	for x := range withdrawals.WithdrawalList {
+		resp = append(resp, exchange.FundHistory{
+			Status:          translateWithdrawalStatus(withdrawals.WithdrawalList[x].Status),
+			Timestamp:       withdrawals.WithdrawalList[x].UpdateTime.Time(),
+			Currency:        withdrawals.WithdrawalList[x].Currency,
+			Amount:          withdrawals.WithdrawalList[x].Amount,
+			TransferType:    "withdrawal",
+			CryptoToAddress: withdrawals.WithdrawalList[x].Address,
+			TransferID:      withdrawals.WithdrawalList[x].TransactionID,
+			Fee:             withdrawals.WithdrawalList[x].Fee,
+		})
+	}
+	for x := range deposits.DepositList {
+		resp = append(resp, exchange.FundHistory{
+			ExchangeName:    cr.Name,
+			Status:          translateDepositStatus(deposits.DepositList[x].Status),
+			Timestamp:       deposits.DepositList[x].UpdateTime.Time(),
+			Currency:        deposits.DepositList[x].Currency,
+			Amount:          deposits.DepositList[x].Amount,
+			TransferType:    "deposit",
+			CryptoToAddress: deposits.DepositList[x].Address,
+			CryptoTxID:      deposits.DepositList[x].ID,
+		})
+	}
+	return resp, nil
+}
+
+// GetWithdrawalsHistory returns previous withdrawals data
+func (cr *Cryptodotcom) GetWithdrawalsHistory(ctx context.Context, c currency.Code, a asset.Item) ([]exchange.WithdrawalHistory, error) {
+	withdrawals, err := cr.GetWithdrawalHistory(ctx)
+	if err != nil {
+		return nil, err
+	}
+	resp := make([]exchange.WithdrawalHistory, len(withdrawals.WithdrawalList))
+	for x := range withdrawals.WithdrawalList {
+		resp[x] = exchange.WithdrawalHistory{
+			Status:          translateWithdrawalStatus(withdrawals.WithdrawalList[x].Status),
+			Timestamp:       withdrawals.WithdrawalList[x].UpdateTime.Time(),
+			Currency:        withdrawals.WithdrawalList[x].Currency,
+			Amount:          withdrawals.WithdrawalList[x].Amount,
+			TransferType:    "withdrawal",
+			CryptoToAddress: withdrawals.WithdrawalList[x].Address,
+			TransferID:      withdrawals.WithdrawalList[x].TransactionID,
+			Fee:             withdrawals.WithdrawalList[x].Fee,
+		}
+	}
+	return resp, nil
+}
+
+// GetRecentTrades returns the most recent trades for a currency and asset
+func (cr *Cryptodotcom) GetRecentTrades(ctx context.Context, p currency.Pair, assetType asset.Item) ([]trade.Data, error) {
```

Comment:
The assetype is not being checked, please run wrapper checks in the cmd folder to see any issues. There are wrapper functions that are not handling formatting pairs.